samcli/lib/build/app_builder.py

```diff
@@ -67,7 +67,7 @@ def __init__(
         docker_client: Optional[docker.DockerClient] = None,
         container_env_var: Optional[Dict] = None,
         container_env_var_file: Optional[str] = None,
-        build_images: Optional[dict] = None,
+        build_images: Optional[Dict] = None,
```

Comment:
Curious: What is the difference between `dict` and `Dict` types?

Reply:
"There is no real difference between using a plain typing.Dict and dict, no.
However, typing.Dict is a Generic type * that lets you specify the type of the keys and values too, making it more flexible" e.g. Dict[str, str]
In general we are just using Dict since it has more functionality. As we already imported Dict and Dict is used everywhere else in the class. Changing this for consistency.

Reply:
Yes, it is just for consistency